apps/mail-bridge/postal-db/functions.ts

```diff
@@ -328,6 +339,18 @@ export async function getDomainDNSRecords(
     }
   }
 
+  const dmarcRecord = await lookupTXT('_dmarc.' + domainInfo.name);
+  if (dmarcRecord.success) {
+    const dmarcValue = dmarcRecord.data.find((_) => _.startsWith('v=DMARC1'));
+    if (dmarcValue) {
+      const policyMatch = dmarcValue.match(/p=([^\s;]+)/);
```

Comment:
We have a Dmrc Parser in utils/dns/txtParsers like spf and dkim parser, use should use that instead of regex.